.circleci/config.yml

```diff
@@ -2,33 +2,31 @@ version: 2
 jobs:
   build:
     docker:
-    - image: circleci/golang:1.9
-
-    - image: circleci/mongo:4.0.0
-
-    working_directory: /go/src/github.com/jamillosantos/http-mgo-service
+      - image: circleci/golang:1.12
+      - image: circleci/mongo:4.0.0
     steps:
       - checkout
-
       - run: sudo apt install mongodb-clients libssl-dev
```

Comment:
I believe that at any given time this library will fail (`libssl-dev`) :sweat:

See you [link](https://circleci.com/gh/lab259/auth/175?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).